tests/test_geom.py

```diff
@@ -98,3 +98,9 @@ def test_get_geom_2d_flow_area_attrs(tmp_path):
     )
     ras_hdf = RasGeomHdf(test_hdf)
     assert ras_hdf.get_geom_2d_flow_area_attrs() == TEST_ATTRS
+
+
+def test_structs():
+    structs_json = TEST_JSON / "structures.json"
+    with RasGeomHdf(MUNCIE_G05) as ghdf:
+        assert _gdf_matches_json(ghdf.structures(), structs_json)
```

Comment:
I think I might handle it like this, updating `structures.json` accordingly:
```python
def test_structs():
    structs_json = TEST_JSON / "structures.json"
    with RasGeomHdf(MUNCIE_G05) as ghdf:
        structures_gdf = ghdf.structures()
        structures_gdf["Last Edited"] = ghdf.structures()["Last Edited"].astype(str)
        assert _gdf_matches_json(structures_gdf, structs_json)
```

Reply:
updated this to leverage optional `datetime_to_str` argument within `structures` method